Modules/FT0/src/DigitQcTask.cxx

```diff
@@ -321,7 +321,7 @@ void DigitQcTask::monitorData(o2::framework::ProcessingContext& ctx)
   for (auto& digit : digits) {
     const auto& vecChData = digit.getBunchChannelData(channels);
     bool isTCM = true;
-    if (digit.mTriggers.amplA == -5000 /*&& digit.mTriggers.amplC == -5000 && digit.mTriggers.timeA == -5000 && digit.mTriggers.timeC == -5000*/) {
+    if (digit.mTriggers.timeA == -5000 && digit.mTriggers.timeC == -5000) {
```

Comment:
Is there any particular reason why you used hardcoded value `-5000` instead of the `o2::fit::Triggers::DEFAULT_TIME`

Reply:
> Is there any particular reason why you used hardcoded value `-5000` instead of the `o2::fit::Triggers::DEFAULT_TIME`

Hi @maciej, yes , hotfix w/o compiling :)